src/modules/item/starred/__test__/delete.test.ts

```diff
@@ -0,0 +1,168 @@
+import { User } from '@prisma/client';
+import UserService from '../../../auth/user.service';
+import AuthService from '../../../auth/auth.service';
+import StarredService from '../starred.service';
+import FolderService from '../../folder/folder.service';
+
+describe('DELETE /api/starred/:id', () => {
+	let userService: UserService;
+	let authService: AuthService;
+	let starredService: StarredService;
+	let folderService: FolderService;
+
+	let user: User;
+	let otherUser: User;
+
+	beforeAll(async () => {
+		authService = new AuthService();
+		userService = new UserService();
+		starredService = new StarredService();
+		folderService = new FolderService();
+
+		user = await userService.createUser({
+			name: 'Joe Biden the 1st',
+			email: 'joe@biden.com',
+			password: '1234',
+		});
+		otherUser = await userService.createUser({
+			name: 'Joe Biden the 2nd',
+			email: 'joe2@biden.com',
+			password: '4321',
+		});
+	});
+
+	it('should delete starred', async () => {
+		const { accessToken } = await authService.createTokens(user.id);
+
+		const folder = await folderService.createFolder({
+			name: 'Folder1',
+			ownerId: user.id,
+			parentId: null,
+			color: '#78BC61',
+		});
+
+		const starred = await starredService.createStarred({
+			itemId: folder.id,
+			userId: user.id,
+		});
+
+		const response = await global.fastify.inject({
+			method: 'DELETE',
+			url: '/api/starred/' + starred.id,
+			headers: {
+				authorization: 'Bearer ' + accessToken,
+			},
+		});
+
+		expect(response.statusCode).toBe(204);
+		expect(response.body).toBe('');
+
+		await expect(starredService.getById(starred.id)).rejects.toThrowError();
+	});
+
+	it('should return status 401, when unauthorized', async () => {
+		const folder = await folderService.createFolder({
+			name: 'Folder1',
+			ownerId: user.id,
+			parentId: null,
+			color: '#78BC61',
+		});
+
+		const starred = await starredService.createStarred({
+			itemId: folder.id,
+			userId: user.id,
+		});
+
+		const response = await global.fastify.inject({
+			method: 'DELETE',
+			url: '/api/starred/' + starred.id,
+			headers: {
+				authorization: 'invalid_access_token!!!',
+			},
+		});
+
+		expect(response.statusCode).toBe(401);
+		expect(response.json()).toEqual({
+			error: 'UnauthorizedError',
+			errors: {
+				_: ['Unauthorized'],
+			},
+			statusCode: 401,
+		});
+	});
+
```

Comment:
Add test making sure you can delete your staring on an item, you don't have access to